pkg/manifest/manifest.go

```diff
@@ -141,16 +141,3 @@ func (e Environments) FilterByNames(names []string) (Environments, error) {
 
 	return result, nil
 }
-
-// FilterByGroup returns all environments whose group-name matches the given name.
-func (e Environments) FilterByGroup(groupName string) Environments {
```

Comment:
commit nitpick: why is it removed? (In other commits you state 'unused function' in the message)

Reply:
reworded